<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkpSocketTestWidget</class>
 <widget class="QWidget" name="SkpSocketTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>socketTimer</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="startTimer">
          <property name="text">
           <string>startTimer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopTimer">
          <property name="text">
           <string>stopTimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="socketTimerOnce">
       <property name="title">
        <string>socketTimerOnce</string>
       </property>
       <widget class="QPushButton" name="timerOnce">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>timerOnce</string>
        </property>
       </widget>
       <widget class="QPushButton" name="protocolConn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>protocolConn</string>
        </property>
       </widget>
       <widget class="QPushButton" name="protocolConnClose">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>protocolConnClose</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="connectOnce">
       <property name="title">
        <string>connectOnce</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="connectSocket">
          <property name="text">
           <string>connectSocket</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="socketSend">
          <property name="text">
           <string>socketSend</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="socketRecv">
          <property name="text">
           <string>socketRecv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="socketClose">
          <property name="text">
           <string>socketClose</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="timerTaskStart">
          <property name="text">
           <string>timerTaskStart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="onceSocketTaskStart">
          <property name="text">
           <string>onceSocketTaskStart</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="autoConnect_label">
              <property name="text">
               <string>autoConnect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="autoconnect_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="showPrint_label">
              <property name="text">
               <string>showPrint</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="showPrint_lineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="ok_pushButton">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>154</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
